plugin/src/main/java/io/snyk/eclipse/plugin/views/snyktoolview/SnykToolView.java

```diff
@@ -252,7 +268,46 @@ public void toggleIgnoresButtons() {
 			menuManager.update(true);
 
 		});
+	}
+
+	/*
+	 * Sets up for doing a Net New Issues scan.
+	 */
+	public void enableDelta() {
+		if (this.treeViewer != null && !this.treeViewer.getTree().isDisposed()) {
+			BaseTreeNode[] children = (BaseTreeNode[]) getRoot().getChildren();
+
+			for (BaseTreeNode node : children) {
+				if (node instanceof ContentRootNode) {
+					ContentRootNode contentNode = (ContentRootNode) node;
+					String projectPath = contentNode.getPath().toString();
+					String projectName = ResourceUtils.getProjectByPath(contentNode.getPath()).getName();
+					String baseBranch = folderConfigs.getBaseBranch(projectPath);
+
+					contentNode.setName(String.format("%s - Click here choose base branch [ current: %s ]",
+							projectName, baseBranch));
+				}
+			}
+		}
+	}
+
+	/*
+	 * Disables Net New Issues scan, and starts a regular scan.
+	 */
+	public void disableDelta() {
+		if (this.treeViewer != null && !this.treeViewer.getTree().isDisposed()) {
+			BaseTreeNode[] children = (BaseTreeNode[]) getRoot().getChildren();
+
+			for (BaseTreeNode node : children) {
+				if (node instanceof ContentRootNode) {
+					ContentRootNode contentNode = (ContentRootNode) node;
+					String projectName = ResourceUtils.getProjectByPath(contentNode.getPath()).getName();
+					contentNode.setName(projectName);
+				}
+			}
+		}
 
+		SnykExtendedLanguageClient.getInstance().triggerScan(null);
```

Comment:
if you're in the UI thread, this should be called async, e.g. with CompletableFuture.runAsync...